DEVELOPMENT_ENVIRONMENT.md

```diff
@@ -2,7 +2,7 @@
 # SORMAS Development Environment
 
 ## Server
-- [Install your local server](SERVER_SETUP.md)
+- Install [your local server](SERVER_SETUP.md) or [a more dev specific one](SERVER_DEV_SETUP.md)
```

Comment:
Maybe make it clear here that the dev-specific guide is based on using Docker?

Reply:
Done